internal/plugins/envtest/init.go

```diff
@@ -61,6 +61,8 @@ func initUpdateMakefile(filePath string) error {
 }
 
 const makefileTestTarget = `# Run tests
+# Set default shell as bash
+SHELL := /bin/bash
```

Comment:
Based on the default Ubuntu installation `/bin/sh` is a symlink to `dash` a different shell. Changing `source SCRIPT` to `. SCRIPT` would work but that can't be done in the SDK since it is part of the kubebuilder scaffolding.  Since we can't really change `source` to `.` in a timely fashion, setting `SHELL` is fine.